### PR TITLE
Removed unnecessary exception for no world plugins

### PR DESCRIPTION
**Pull request by thehutch**
Signed-off-by:thehutch

### CODE REVIEW

File: src/main/java/org/spout/engine/SpoutEngine.java [hunk 1]
```diff
@@ -853,6 +854,7 @@ public CompletionManager getCompletionManager() {
 		return completions;
 	}
```
Comment (windy1) [inline src/main/java/org/spout/engine/SpoutEngine.java:855]: I don't think this is any better to be honest, this is the kind of place where an IllegalStateException is appropriate.

Comment (thehutch) [inline src/main/java/org/spout/engine/SpoutEngine.java:855]: Except having a stacktrace is completely pointless, that's why I removed it. Yes, it's in a illegal state but a stacktrace isn't necessary just a simple log message

Comment (kitskub) [inline src/main/java/org/spout/engine/SpoutEngine.java:855]: Agreed.

Comment (dredhorse) [inline src/main/java/org/spout/engine/SpoutEngine.java:855]: seconded

**Pull request closed without merging**